From d4c4ea8c4ea77471fc72639d31c38d3d39bc0453 Mon Sep 17 00:00:00 2001
From: Peter Robinson <pbrobinson@gmail.com>
Date: Wed, 23 Jan 2019 03:19:21 +0000
Subject: [PATCH 06/13] Setup graphical console on all architectures except
 s390x

The graphical console is now supported on all architectures except
s390x so set it up to enable screenshot fucntionality across all
supported architectures.

Signed-off-by: Peter Robinson <pbrobinson@gmail.com>
---
 oz/Guest.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/oz/Guest.py b/oz/Guest.py
index 418b888..dfb63e4 100644
--- a/oz/Guest.py
+++ b/oz/Guest.py
@@ -500,8 +500,8 @@ class Guest(object):
         # devices
         devices = oz.ozutil.lxml_subelement(domain, "devices")
         # graphics
-        if self.tdl.arch not in ["aarch64", "armv7l", "s390x"]:
-            # qemu for arm/aarch64/s390x does not support a graphical console - amazingly
+        if not self.tdl.arch in ["s390x"]:
+            # qemu for s390x does not support a graphical console
             oz.ozutil.lxml_subelement(devices, "graphics", None, {'port': '-1', 'type': 'vnc'})
         # network
         interface = oz.ozutil.lxml_subelement(devices, "interface", None, {'type': 'bridge'})
-- 
2.20.1

